CMakeLists.txt

```diff
@@ -362,49 +307,49 @@ else()
   message(FATAL_ERROR "Could not find python3 to use in qt/, shaders/ and 3party/.")
 endif()
 
-add_subdirectory(base)
-add_subdirectory(coding)
-add_subdirectory(descriptions)
-add_subdirectory(drape)
-add_subdirectory(drape_frontend)
-add_subdirectory(editor)
-add_subdirectory(ge0)
-add_subdirectory(generator/mwm_diff)
-add_subdirectory(geometry)
-add_subdirectory(indexer)
-add_subdirectory(kml)
-add_subdirectory(map)
-add_subdirectory(cppjansson)
-add_subdirectory(platform)
-add_subdirectory(routing)
-add_subdirectory(routing_common)
-add_subdirectory(search)
-add_subdirectory(shaders)
-add_subdirectory(storage)
-add_subdirectory(tracking)
-add_subdirectory(traffic)
-add_subdirectory(transit)
+omim_add_subdirectory("${PROJECT_NAME}" base)
```

Comment:
What doesn't work if we keep add_subdirectory() original?

Reply:
Check the answer above